Gemfile

```diff
@@ -11,7 +11,7 @@ end
 
 alchemy_version = ENV.fetch("ALCHEMY_VERSION", "main")
 if alchemy_version == "main"
-  gem "alchemy_cms", github: "AlchemyCMS/alchemy_cms", branch: "main"
+  gem "alchemy_cms", github: "AlchemyCMS/alchemy_cms", branch: "admin_importmaps"
```

Comment:
We can now do `main`! (For when you rebase the conflict in the `ci.yml`).

Reply:
done